compiler/compile/OMRCompilation.hpp

```diff
@@ -575,6 +575,8 @@ class OMR_EXTENSIBLE Compilation
 
    TR::list<TR::Snippet*> *getSnippetsToBePatchedOnClassRedefinition();
 
+   TR::Block *insertNewFirstBlock();
```

Comment:
This is a new API.  Please create Doxygen comment describing the API and what it does.

Reply:
Done.